torch/nn/modules/loss.py

```diff
@@ -449,6 +449,8 @@ class KLDivLoss(_Loss):
     Examples::
 
         >>> import torch.nn.functional as F
+        >>> from torch import nn
+        >>> import torch
```

Comment:
should maybe `import torch` be moved above `import torch.nn.functional as F`? also, style of `from torch import nn` does not match the style of `import torch.nn.functional as F`

I would propose (or at least make them the same style):
```
import torch
import torch.nn as nn
import torch.nn.functional as F
```

Reply:
sure